applications/mne_analyze/mne_analyze/analyzecore.cpp

```diff
@@ -169,7 +169,7 @@ void AnalyzeCore::initMainWindow()
 {
     m_pMainWindow = new MainWindow(m_pPluginManager);
     QObject::connect(m_pMainWindow.data(), &MainWindow::mainWindowClosed,
-                     this, &AnalyzeCore::onMainWindowClosed);
+                     this, &AnalyzeCore::onMainWindowClosed, Qt::DirectConnection);
```

Comment:
Why not AutoConnection? Are the two objects running in different threads?

Reply:
I was testing to see if the inconsistent destructor calling was due to the analyzecore onWindowClosed being called at different times. It wasn't, and I forgot to change it back. Will remove.